lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -674,6 +684,10 @@ export class AuthorizationCodeClient extends BaseClient {
             );
         }
 
+        if (request.brokerParameters) {
+            parameterBuilder.addBrokerParameters(request.brokerParameters);
+        }
+
         this.addExtraQueryParams(request, parameterBuilder);
```

Comment:
Isn't this duplicated code if a few params are added above?

Reply:
Added tests.